nikola/nikola.py

```diff
@@ -28,7 +28,6 @@
 
 from __future__ import print_function, unicode_literals
 import io
-from collections import defaultdict
```

Comment:
I fixed it and I’m currently testing this. Will merge if all works okay (which it currently does)